### PR TITLE
Lint:Ensure multi-line arguments align close to left margin

### PR DESCRIPTION
**Pull request by marcotc**
For my friend, @ivoanjo ❤️ 

This PR reduces the indentation level of multi-line arguments.

From:
```ruby
      expect(build_location.to_h).to match(hash_including(id: location_id,
                                                          line: [{
                                                            function_id: function.id, line: line_number
                                                          }]))
```
To:
```ruby
      expect(build_location.to_h).to match(hash_including(id: location_id,
        line: [{
          function_id: function.id, line: line_number
        }]))
```

2587 white-spaces removed 🙂:
```ruby
git diff head^ | ruby -e "puts ARGF.readlines.select{|x| x.start_with?('-', '+')}.map{|x| s = x.scan(' ').size; x.start_with?('-') ? -s : s}.sum"
# -2587
```

### Follow up

One issue this change has is that it leaves the first parameter in a muiti-line argument block alone, at the top-right side:
```ruby
      expect(build_location.to_h).to match(hash_including(id: location_id,
        line: [{                                        # ^^ This one
          function_id: function.id, line: line_number
        }]))
```

```ruby
                @_response = ::ActionDispatch::Response.new(403,
                  { 'Content-Type' => 'text/html' },      # ^^ This one
                  [Datadog::AppSec::Assets.blocked])
```

This looks a bit off, and hard to scan for all parameters.
I chose to not address that in this PR, because that required adding 5 new cops just to make it not look crazy, so a follow up PR does that work: #2166
Also, that work touches way more files.

### CODE REVIEW

**Review by ivoanjo — APPROVED** [review]
👍 Thank you for improving this! :)

One tiny note -- I learned the other day that git now allows you to create a `.git-blame-ignore-revs` file that lists commits which are ignored for blame purposes... and that GitHub even supports it now!

https://docs.github.com/en/repositories/working-with-files/using-files/viewing-a-file#ignore-commits-in-the-blame-view

TL;DR: I think this PR/commit is a good candidate for our first use of such a feature, waddayouthink?
